### PR TITLE
update hyperparameters of bayeisan net

### PR DESCRIPTION
**Pull request by atinary-rhickman**
Small changes to the hyperparameters of the bayesian network. I noticed we were sampling duplicate parameters with the multiple sampling strategies, which is a result of `nans` generated in the calculation resulting in the rewards being the same. 

Both of these param changes smooth out the continuous approximation to the categorical dist and removes this issue.

### CODE REVIEW

File: src/gryffin/bayesian_network/model_details.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 #========================================================================
 
 model_details = {
-                'num_epochs':  2 * 10**3, 'learning_rate': 0.05,
+                'num_epochs':  2 * 10**3, 'learning_rate': 0.01,
```
Comment (atinary-jruza) [inline src/gryffin/bayesian_network/model_details.py:8]: Where do the new values for both of these come from?
Actually, where do the old ones come from as well :D

Comment (atinary-rhickman) [reply src/gryffin/bayesian_network/model_details.py:8]: The new learning rate comes from my tests. With a smaller learning rate I dont see any parameters duplication. Im not sure why the original learning rate is set this high, might just be copied from Phoenics

**Review by atinary-aduran — APPROVED** [review]

File: src/gryffin/bayesian_network/tfprob_interface/tfprob_interface.py [hunk 2]
```diff
@@ -220,7 +220,8 @@ def construct_model(self):
 
                     target = tf.cast(self.y[:, kernel_begin : kernel_end], tf.int32)
 
-                    prior_temperature = 0.5 + 10.0 / self.num_obs
+#                    prior_temperature = 0.5 + 10.0 / self.num_obs
```
Comment (atinary-screoff) [inline src/gryffin/bayesian_network/tfprob_interface/tfprob_interface.py]: can you drop old code please ?

**Review by atinary-screoff — CHANGES_REQUESTED** [review]

File: src/gryffin/bayesian_network/tfprob_interface/tfprob_interface.py (hunk 2 above)
Comment (atinary-rhickman) [reply src/gryffin/bayesian_network/tfprob_interface/tfprob_interface.py]: done

**Review by atinary-jruza — APPROVED** [review]
